src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incidents/IncidentResourceIntTest.kt

```diff
@@ -55,29 +69,37 @@ class IncidentResourceIntTest : IntegrationTestBase() {
       responseRecordingStaff = staff(firstName = "ALBERT", lastName = "STAFF") {
         account(username = "ALBERTSTAFF")
       }
-
-      lateinit var question1: QuestionnaireQuestion
-      lateinit var question2: QuestionnaireQuestion
-      lateinit var question2Answer1: QuestionnaireAnswer
-      lateinit var question3: QuestionnaireQuestion
-      lateinit var question3Answer1: QuestionnaireAnswer
-      lateinit var question3Answer3: QuestionnaireAnswer
       questionnaire1 = questionnaire(code = "ESCAPE_EST", description = "Escape Questionnaire") {
         val question4 = questionnaireQuestion(question = "Q4: Any Damage amount?") {
           questionnaireAnswer(answer = "Q4A1: Enter Damage Amount in Pounds")
```

Comment:
A lot of config is needed to set up this data - it is starting to look quite busy/noisy (and this was noted in a previous PR).  I will see if I can tidy this up in a separate PR.